src/styles/main.scss

```diff
@@ -1 +1,53 @@
-// put code here
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stopwatch {
+  display: flex;
+  position: absolute;
+  width: 300px;
```

Comment:
The width and height of the '.stopwatch' class is static. It would be better to use dynamic sizing like percentages or viewport units. As per the requirement, the size should be 80vmin.